### PR TITLE
Translate language ISO codes into names when displaying

### PR DESCRIPTION
**Pull request by rowanseymour**
Also adds language to the contact details box on the case read page:

screen shot 2016-07-22 at 09 38 23

### CODE REVIEW

File: casepro/utils/__init__.py [hunk 1]
```diff
@@ -164,3 +168,23 @@ def uuid_to_int(uuid):
     "Values from -2147483648 to 2147483647 are safe in all databases supported by Django"
     """
     return UUID(hex=uuid).int % (2147483647 + 1)
+
+
+def get_language_name(iso_code):
+    """
+    Gets a language name for a given ISO639-2 code.
```
Comment (rowanseymour) [inline casepro/utils/__init__.py]: RapidPro uses ISO639-2 for language - if other deployments use a different code set then maybe do this translation in the backend abstraction.

File: casepro/contacts/models.py [hunk 2]
```diff
@@ -269,6 +270,11 @@ def as_json(self, full=True):
         if full:
             result['fields'] = self.get_fields(visible=True)
 
+            if self.language:
```
Comment (norkans7) [inline casepro/contacts/models.py]: may be a method on the model, and reuse that in the views as well

File: casepro/contacts/views.py [hunk 3]
```diff
@@ -44,6 +44,9 @@ def derive_fields(self):
 
             return fields
 
+        def get_language(self, obj):
+            return get_language_name(obj.language) if obj.language else None
```
Comment (norkans7) [inline casepro/contacts/views.py]: here use the model method

Comment (rowanseymour) [inline casepro/contacts/views.py]: This entire view is about to replaced in the next PR

**Comment by norkans7** [comment]
Yep

**Pull request merged**